tst/testlibgap/0001.c

```diff
@@ -0,0 +1,57 @@
+/*
+ * Small program to test libgap linkability and basic working
+ */
+#include <stdio.h>
+#include <unistd.h>
+
+#include <compiled.h>
+#include <libgap-api.h>
+
+extern char ** environ;
+
+void test_eval(const char * cmd)
+{
+    Obj res, ires;
+    Int rc, i;
+
+    printf("Input:\n%s", cmd);
+    res = GAP_EvalString(cmd);
+    rc = GAP_LenPList(res);
+    for(i=1;i<=rc;i++) {
+        ires = GAP_ElmPList(res, i);
+        if( GAP_ElmPList(ires, 1) == True )
+            printf("Output:\n%s\n", CSTR_STRING(GAP_ElmPList(ires, 5)));
+    }
```

Comment:
Would it perhaps be helpful to print the output in the same format as a `.tst` file?
(Fine by me to leave it as is, just wondering)